src/components/pages/devices/flyouts/deviceDetails/deviceDetails.scss

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/mixins';
+@import 'src/styles/themes';
+
+.device-details-container {
+  display: flex;
+  flex-flow: column nowrap;
+
+  .form-section {
+
+    .section-header {
+      border-top-style: solid;
+      border-top-width: 1px;
+      padding: 0.25rem 0 0 0;
+    }
+
+    .section-content { padding: 0 1rem 0 1rem; }
+
+    .property-row {
+      display: flex;
+
+      .property-name {
+        padding-inline-end: 0.5rem;
+        width: 30% !important;
```

Comment:
My "column" alignment wasn't working.  At some point it did work... but may have been unrelated to adding !important.  Since it seems to work now without it.